tests/runtime_neutral/test_runtime_delivery_acceptance.py

```
@@ -414,6 +426,525 @@ def test_runtime_delivery_acceptance_fails_when_disclosure_contradicts_approved_
+        self.assertEqual(1, report["execution_context"]["specialist_host_executed_unit_count"])
+        self.assertEqual("host_current_session_executed", report["execution_context"]["specialist_effective_execution_status"])
+        self.assertIn(
+            str(session_root / "specialist-execution.json"),
+            report["truth_results"]["workflow_completion_truth"]["evidence"],
+        )
+
+    def test_runtime_delivery_acceptance_does_not_pass_incomplete_workflow_with_executed_sidecar(self) -> None:
+        approved_dispatch = [
+            {
+                "skill_id": "systematic-debugging",
+                "native_skill_entrypoint": "/tmp/systematic-debugging/SKILL.md",
```

Ruff (0.15.11)

[error] Probable insecure usage of temporary file or directory: "/tmp/systematic-debugging/SKILL.md"

(S108)